spec/api-media-handler-spec.ts

```diff
@@ -243,6 +243,29 @@ ifdescribe(features.isDesktopCapturerEnabled())('setDisplayMediaRequestHandler',
     expect(ok).to.be.true(message);
   });
 
+  it('can supply a frame response for audio with disableLocalEcho set to false', async () => {
+    const ses = session.fromPartition('' + Math.random());
+    let requestHandlerCalled = false;
+    ses.setDisplayMediaRequestHandler((request, callback) => {
+      requestHandlerCalled = true;
+      callback({
+        video: w.webContents.mainFrame,
+        audio: w.webContents.mainFrame,
+        disableLocalEcho: false
+      });
+    });
+    const w = new BrowserWindow({ show: false, webPreferences: { session: ses } });
+    await w.loadURL(serverUrl);
+    const { ok } = await w.webContents.executeJavaScript(`
+      navigator.mediaDevices.getDisplayMedia({
+        video: true,
+        audio: true,
+      }).then(x => ({ok: x instanceof MediaStream}), e => ({ok: false, message: e.message}))
+    `, true);
+    expect(requestHandlerCalled).to.be.true();
+    expect(ok).to.be.true();
+  });
```

Comment:
I think this test would pass even if your code changes weren't applied, which makes it a bit questionably useful. Is there any easy way to check if local echo has been enabled? is that a property you can check anywhere?

If not, it may simply not be worth having a test for this.

Reply:
Not to my knowledge. I'll remove the test.